frameworks/bf/run-bf.js

```diff
@@ -0,0 +1,7 @@
+var execSync = require('child_process').execSync;
+module.exports.runBf = function(input) {
```

Comment:
`execSync` returns `Buffer` by default and that's a string representation of `Buffer`. Nothing to do with `bf`.

```js
console.log(new Buffer("Hello World!"))
//=> <Buffer 48 65 6c 6c 6f 20 57 6f 72 6c 64 21>

console.log(new Buffer("Hello World!").toString('utf8'))
console.log(new Buffer("Hello World!").toString())
//=> Hello World!
```